examples/JsEventHandlerExamples/multi-field-manipulation.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Name: Field Manipulation Event Handlers
+ * Description: This is a set of event handlers that demonstrate the various methods to manipulate fields.
```

Comment:
I don't remember the context around why this example was written but it feels like it could use more background info or some example scenarios for when this would be useful.

Reply:
Looks good to me, thanks!